README.md

```diff
@@ -56,6 +45,28 @@ See the [documentation](http://pm2.keymetrics.io/docs/usage/docker-pm2-nodejs/#u
 }
 ```
 
+### Create a `Dockerfile` in your Node.js app project
+Assuming that `pm2.json` and `package.json` are in the same folder of the `Dockerfile` and that the `src` folder contains your source code.
+
+```dockerfile
+FROM keymetrics/pm2-docker-alpine:latest
+
+# Bundle APP files
+COPY src src/
+COPY package.json .
+COPY pm2.json .
```

Comment:
Something like that?
https://github.com/keymetrics/docker-pm2/blob/b67352879099f165ae06750a829665a4536366da/README.md#create-a-pm2-process-file-in-your-nodejs-app-project